build.gradle

```diff
@@ -13,16 +13,24 @@ buildscript {
     }
     mavenCentral()
   }
+  repositories {
+    maven {
+      url "https://plugins.gradle.org/m2/"
+    }
+  }
```

Comment:
We actually declare the block 3 times :smile: under different circumstances. This one is scoped under buildscript, the others are not.